data/Text.csv

```diff
@@ -1,4 +1,4 @@
-text_id,title,subtitle,title_lang,person_id,genre_id,publication_place,publication_date,publishing_house,first_chin_edition,physical_description,neibu,source_id,page_number,note,created,created_by,last_modified,last_modified_by
+text_id,title,subtitle,title_lang,person_id,genre_id,publication_place,publication_date,publishing_house,first_chin_edition,physical_description,neibu,source_id,pages,note,created,created_by,last_modified,last_modified_by
```

Comment:
first_chin_edition is still in file see #31 ?

Reply:
Yeah, I thought we are about to keep it as an additional information?